schema/individual.json

```diff
@@ -12,7 +12,7 @@
       "type": "array",
       "items": {
         "type": "string",
-        "enum": ["ECMA-262", "ECMA-402", "inactive", "withdrawn", "archived"]
+        "enum": ["ECMA-262", "ECMA-402", "inactive", "withdrawn", "deprecated", "archived"]
```

Comment:
the `deprecated` definition, i reference `compat-data` in `status_block`

see https://github.com/mdn/browser-compat-data/blob/bd8f3ed92d3c13aebfa92ac69d71ef7ed2441e71/schemas/compat-data.schema.json#L117-L135

Reply:
ah, this might be a bit confusing in our case... it is much rarer in ecmascript that we deprecate something and we haven't used that language yet. We do have a current removal attempt in progress, but this will result in a break and we are not yet sure how to handle it (see this [issue](https://github.com/tc39/Reflector/issues/339)). So this doesn't apply in the same way to us as to other standards.

I am on the fence with this tag, maybe let's leave it out?

Reply:
e.g: `__proto__` in mdn, https://mdn.io/__proto__

Reply:
@Jack-Works any idea?

Reply:
Yes, annex b overlaps with deprecated features. I guess the difference is in how use of a given feature is communicated vs how the spec thinks of them. Cases such as __proto__ are normative optional. Last meeting there was [further discussion](https://github.com/tc39/notes/blob/caf460afbda8588d757a3c40c06b2dc29fe4148e/meetings/2020-11/nov-16.md#proto-normative-optional-options) to move more into the main spec. Indeed, there was [discussion around much of annex b](https://github.com/tc39/notes/blob/827c5f98554b6e04d81c2c265f78f6f7ddf8415a/meetings/2019-06/june-4.md#annex-b-reform )(unfortunately we don't have the slides right now). However we may consider some features deprecated from a user stand point, but all browsers support it, as annex b is a requirement for browser compatibility. but it isn't something we have used as a language in the committee as far as I know. 

To moving all of annex b, there was also a counter argument to this from waldemar: https://docs.google.com/presentation/d/e/2PACX-1vRPvLtLI7ln2yVRm2wYU2j3ByAJwpDjQmJSS4okYBnWN8OgTnZk7kFz5t2ARvpIEjUg_rG_aYVEcANr/pub?start=false&loop=false&delayms=3000#slide=id.g635b95875a_0_0

So this is where my hesitation comes from. We have removed things from the specification. But we are not clear in our language about how we do it. 

I guess, to answer this is we need to look at a different question -- how and where would we use this tag?